acmwebsite/lib/helpers.py

```diff
@@ -29,6 +29,10 @@ def icon(icon_name):
     return Markup('<i class="glyphicon glyphicon-%s"></i>' % icon_name)
 
 
+def fa_icon(icon_name):
+    return Markup('<i class="fa fa-%s"></i>' % icon_name)
```

Comment:
Don't use `printf` style formatting. The `str.format` function available since Python 2.5 is a better choice.